### ISSUE TITLE
Domanda: come disabilitare la possibilità di creare utenti?

### ISSUE THREAD

**Issue by cloudify** [italia/spid-testenv2#157]
Ciao,

ho impostato degli utenti tramite `users.json` e vorrei disabilitare la possibilità di crearne nuovi poiché il deployment e' pubblico, come posso fare?

Ci sono altri accorgimenti di sicurezza in caso di deploy pubblici?

grazie

**Comment by alexrj** [comment]
Mi sembra una giusta feature request... al momento non mi risulta che si possa.

**Comment by fmarco** [comment]
@alexrj @cloudify Esatto, al momento non è possibile disabilitare l'aggiunta di utenti.
Potremmo aggiungere un'opzione nella configurazione

**Comment by cloudify** [comment]
@fmarco si grazie sarebbe molto utile, noi abbiamo un deploy pubblico e meno funzionalita' sono esposte meglio e' :)
